paper-ieee-trans.tex

```diff
@@ -346,23 +348,23 @@ \subsection{INtegrating Distributed data Infrastructures for Global
 \section{New trends in software reliability}
 \label{sec:ntsr}
 
-{\sl INDIGO--DataCloud} is the most recent project included in this paper thus it
+{\sl INDIGO--DataCloud} is the most recent project included in this paper thus, it
```

Comment:
subsitute with "the most recent project included in this paper and thus leverage the lessons learned .."

Reply:
ok